### ISSUE TITLE
[Bug] remove <version> - removing symlink as dir

### ISSUE THREAD

**Issue by fzzr-** [HaxeFoundation/haxelib#202]
```log
$ haxelib remove nme-dev 1.3.2
Error rmdir /Users/ak/Developer/Haxe/libs/nme-dev/1,3,2/project/unpack/SDL2_mixer-2.0.0/Xcode/Frameworks/FLAC.framework/Headers
```

Here `Headers@` & `Resources@` is a symlinks:
```shell
$ cd /Users/ak/Developer/Haxe/libs/nme-dev/1,3,2/project/unpack/SDL2_mixer-2.0.0/Xcode/Frameworks/FLAC.framework/
FLAC.framework ak$ ls -GFAC
Headers@          LICENSE.FLAC.txt  Resources@        Versions/
```

**Comment by fzzr-** [comment]
@back2dos : You can assign this issue to me :)

**Comment by fzzr-** [comment]
Fixed in https://github.com/HaxeFoundation/haxelib/pull/203.

**Comment by back2dos** [comment]
Fixed by #203
